src/main/java/com/redhat/pantheon/servlet/AsciidocRenderingServlet.java

```diff
@@ -44,40 +43,25 @@
 import java.io.IOException;
 import java.io.Writer;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
-import java.util.stream.Collectors;
 
 import static java.util.Collections.singletonList;
 import static java.util.stream.Collectors.toList;
 
-/**
- * Hello World Servlet registered by resource type
- *
- * Annotations below are short version of:
- * 
- * @Component
- * @Service(Servlet.class)
- * @Properties({
- *    @Property(name="service.description", value="Hello World Type Servlet"),
- *    @Property(name="service.vendor", value="The Apache Software Foundation"),
- *    @Property(name="sling.servlet.resourceTypes", value="sling/servlet/default"),
- *    @Property(name="sling.servlet.selectors", value="hello"),
- *    @Property(name="sling.servlet.extensions", value="html")
- * })
- */
-@Component(
-        service = Servlet.class,
-        property = {
-                "sling.servlet.resourceTypes=pantheon/modules",
-                "sling.servlet.extensions=preview",
-                Constants.SERVICE_DESCRIPTION+"=Servlet which transforms asciidoc content into html",
-                Constants.SERVICE_VENDOR+"=Red Hat Content Tooling team"
-        })
+
+@Component(service = { Servlet.class })
+@SlingServletResourceTypes(
```

Comment:
:+1: here. I was surprised to see how this annotation had not been deprecated as a lot of them were when moving over to the OSGI-specific ones.

Reply:
https://sling.apache.org/documentation/the-sling-engine/servlets.html

I'm not sure how to read your comment about deprecation, but far from being deprecated, this is actually the preferred way to register the servlet according to the doc I linked.

Reply:
When I first generated the project, it came with a few servlet examples. These examples used a handful of now-deprecated annotations. I replaced all those with standard osgi annotations without realizing there were some still-active Sling-specific ones. So, the comment just some praise as I learned something new :smiley: